### PR TITLE
Modify repository

### PR DESCRIPTION
**Pull request by damir-abdulin**

### CODE REVIEW

File: example/android/app/src/main/kotlin/com/example/example/MainActivity.kt [hunk 1]
```diff
@@ -1,6 +0,0 @@
-package com.example.example
-
-import io.flutter.embedding.android.FlutterActivity
-
-class MainActivity: FlutterActivity() {
```
Comment (KirillSergeevich) [inline example/android/app/src/main/kotlin/com/example/example/MainActivity.kt:5]: why did you delete an example activity?

Comment (damir-abdulin) [reply example/android/app/src/main/kotlin/com/example/example/MainActivity.kt:5]: Because it's code ruins codefactor and without it app works correctly
